src/lib/connection.ts

```diff
@@ -50,6 +54,7 @@ export interface Connection {
     chain?: ChainName;
     contract: string;
     chainId: number;
+    rpcRelay?: boolean;
```

Comment:
Tried to keep the option kind of short since that is the style we've been following, but no strong opinion here